### ISSUE TITLE
Installation on Windows not working

### ISSUE THREAD

**Issue by patudom** [cosmicds/cosmicds#8]
On some Windows computers, the installation runs with no errors, but "cosmicds hubble" does nothing. (On another Windows computer, the browser opened but had a 404 error.)

@nmearl, we had asked @pkgw about this earlier, so he is also looking into possible causes. I wanted to make sure you are both on the same page and not duplicating efforts. Thanks!

**Comment by pkgw** [comment]
CC @Carifio24 too.

**Comment by Carifio24** [comment]
So I was able to get this working on my Windows machine - in my case, the problem was with Python not being able to access the Win32 API. I've detailed what I did below - I'm curious if this works for anyone else.

```
## All of the commands are run in the Anaconda Prompt ##

# Create and activate a new conda environment
conda create --name cosmicds
activate cosmicds

# Install pip
conda install pip

# I had issues with installing bottleneck and reproject, so I had to grab those separately from the cosmicds package
conda install bottleneck
conda install -c astropy reproject

# Install the CosmicDS package
pip install git+https://github.com/glue-viz/cosmicds

# When I tried to run voila, I got "ImportError: DLL load failed while importing win32api: The specified module could not be found."
# To fix that, install pywin32
conda install pywin32

# Run cosmicds
cosmicds hubble
```

**Comment by pkgw** [comment]
@heywooddogwood are you able to give Jon's recipe a try?

**Comment by heywooddogwood** [comment]
@pkgw @Carifio24 yea I ran through all of Jon's code in an Anaconda Prompt, one line after another. They all worked, and the `pip install` of the CosmicDS package installed a ton of packages (I think this is what you were referring to earlier Peter), with one exception: fast-histogram (PEP 517) errored out. But when I ran `cosmicds hubble`, it spat out `'cosmicds' is not recognized as an internal or external command, operable program or batch file.` I tried navigating into the cosmicds folder in my local GitHub folder, got the same output. Not sure what to try next.
